src/main/app/src/pages/HelpPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import '../css/home.css';
+
+import info from '../constants/keys.js';
+
+// Import Bootstrap CSS and JS. Only this page uses these. 
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/js/bootstrap.js';
+
+// This is a stateless, functional React component used to render each resource in a card format. 
+const ResourceCard = ({ organization, description, url }) => (
```

Comment:
You should try to avoid putting two components in the same file. Everyone component should have its own file. You can put this ResourceCard component in the components folder since it's a helper.

Reply:
Done.